ts/src/pro/woo.ts

```diff
@@ -90,33 +90,49 @@ export default class woo extends wooRest {
         /**
          * @method
          * @name woo#watchOrderBook
-         * @see https://docs.woo.org/#orderbook
+         * @see https://docs.woo.org/#orderbookupdate
          * @description watches information on open orders with bid (buy) and ask (sell) prices, volumes and other data
          * @param {string} symbol unified symbol of the market to fetch the order book for
          * @param {int} [limit] the maximum amount of order book entries to return.
          * @param {object} [params] extra parameters specific to the exchange API endpoint
+         * @param {string} [params.method] either (default) 'orderbook' or 'orderbookupdate'
          * @returns {object} A dictionary of [order book structures]{@link https://docs.ccxt.com/#/?id=order-book-structure} indexed by market symbols
          */
         await this.loadMarkets ();
-        const name = 'orderbook';
+        const defaultMethod = 'orderbook';
+        const method = this.safeString (params, 'method', defaultMethod);
```

Comment:
@sc0Vu next time, please use `handleOptionAndParams` to read this

Reply:
No problem!